### ISSUE TITLE
Show export address and name pointer table entries as pointer to function and names respectively

### ISSUE THREAD

**Issue by op2786** [Vector35/binaryninja-api#3928]
Instead of seeing rva, it would be better to see pointers to functions and function names in those tables.

![export_table](https://user-images.githubusercontent.com/109607896/220839221-0ed65fc0-6d4d-4241-a656-1e28614a2d68.png)

**Comment by plafosse** [comment]
Yes the code needed to change that is right here: https://github.com/Vector35/view-pe/blob/f7d405b56ed0b2944e1e46f94a861317f7949438/peview.cpp#L2018

**Comment by plafosse** [comment]
Oh wait those are RVAs which our type system doesn't really support right now.
